hash/apple2_flop_clcracked.xml

```diff
@@ -49584,4 +48700,1980 @@ license:CC0-1.0
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-06-28-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs graph 1983-01-12 (4am crack).dsk" size="143360" crc="56ee1260" sha1="29f14fd0564e271a10db7c73918735e9149d5d45"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsg840131">
+		<description>pfs: graph (version 1984-01-31) (4am crack)</description>
+		<year>1984</year>
+		<publisher>Software Publishing Corporation</publisher>
+		<info name="developer" value="Software Publishing Corporation" />
+		<info name="usage" value="Runs on any Apple //e or later with 64K. Due to programming errors, this version works best on an unenhanced Apple //e." />
+		<info name="version" value="1984-01-31" />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-06-28-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs graph 1984-01-31 (4am crack).dsk" size="143360" crc="d6629d33" sha1="d97550d6308f80cd629cd88c0d3a5f3ce902fd55"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsg840314">
+		<description>pfs: graph (version 1984-03-14) (4am crack)</description>
+		<year>1984</year>
+		<publisher>Software Publishing Corporation</publisher>
+		<info name="developer" value="Software Publishing Corporation" />
+		<info name="usage" value="Runs on any Apple //e or later with 64K." />
+		<info name="version" value="1984-03-14" />
+		<sharedfeat name="compatibility" value="A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-06-29-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs graph 1984-03-14 (4am crack).dsk" size="143360" crc="a64f0d4c" sha1="4a3e779d550c5c4cf73b91b022e57f7f4050a758"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsrb00">
+		<description>pfs: report (version B.00, 1981-01-03) (4am crack)</description>
+		<year>1981</year>
+		<publisher>Software Publishing Corporation</publisher>
+		<info name="developer" value="Software Publishing Corporation" />
+		<info name="usage" value="Runs on any Apple ][+ or later with 64K. Due to programming errors, this version works best without an 80-column card." />
+		<info name="version" value="B.00, 1981-01-03" />
+		<sharedfeat name="compatibility" value="A2P,A2E,A2EE,A2C,A2GS" />
+		<!--Dump released: 2022-06-29-->
+		<!--productivity program-->
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="pfs report 1981-01-03 (vb.00) (4am crack).dsk" size="143360" crc="47f55268" sha1="a67c4ef6e2591bc0600b565c88f00552eecb5a51"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pfsrb02">
+		<description>pfs: report (version B.02) (4am crack)</description>
```

Comment:
Should alternate versions of pfs: report be clones?

Reply:
@cuavas Completed updated the parent-clone relationships.